README.md

```diff
@@ -33,13 +37,13 @@ tutor distro run-extra-commands
 
 Distro plugin manages a set of settings that you can configure, to know how to do that check:
 
-- [How to custumize distro](./docs/how_to_customize_distro.rst)
+- [How to customize distro](./docs/how_to_customize_distro.rst)
```

Comment:
I think will be necessary to modify https://github.com/eduNEXT/tutor-contrib-edunext-distro/blob/master/docs/how_to_customize_distro.rst to add the extra-commands `DISTRO_EXTRA_COMMANDS`

Reply:
done